### DISCUSSION TITLE
Please do you have some good first issues?

### DISCUSSION THREAD

**Discussion by lsyyysky** [ThinkParQ/beegfs#21] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
Add any other context about the feature request here.

**Comment by iamjoemccormick** [comment, marked as answer]
Hi @lsy789 , 

We currently accept contributions to BeeGFS on a case-by-case basis. Historically most of what we accept are patches for issues people discover, so if you're wanting to contribute this is a great place to start! I will caution we cannot guarantee all patches/contributions will be accepted, so I would encourage first opening a bug before spending any significant time trying to develop a fix. Also note even the contributions we do accept often cannot be used exactly as is, and we need to make some modifications to ensure they work consistently across all supported compilers, operating systems, etc.

However going forward we hope to make it [easier to accept contributions](https://www.beegfs.io/c/beegfs-is-now-on-github-what-does-this-mean-for-you/), which includes creating more issues in the public repository, making it easier for people that want to contribute to pick up existing issues. So hearing people like yourself are interested in actively contributing to BeeGFS is great!
